### PR TITLE
[HEAP-9165] HeapIgnore e2e tests

### PR DESCRIPTION
**Pull request by jmtaber129**
Detox tests for HeapIgnore on the TestDriver app.

### CODE REVIEW

File: e2e/heapIgnore.spec.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    db.orm.connection.sharedRedis().flushall(done);
+  });
+
+  before(async () => {
+    await doTestActions();
+    await rnTestUtil.pollForSentinel('HeapIgnore');
+  });
+
+  it('should ignore the interaction', async () => {
+    const { err, res } = await new Promise((resolve, reject) => {
```
Comment (GrandPoohBear) [inline e2e/heapIgnore.spec.js]: Worth a comment saying what you're doing here.  I had to read it a couple of times to make sure I wasn't missing something!

File: e2e/heapIgnore.spec.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+  await expect(element(by.id('totallyIgnored'))).toBeVisible();
+  await element(by.id('totallyIgnored')).tap();
+  await element(by.id('totallyIgnoredHoc')).tap();
+  await element(by.id('allowedInteraction')).tap();
+  await element(by.id('allowedInnerHierarchy')).tap();
+  await element(by.id('allowedAllProps')).tap();
+
+  // :HACK: Break up long URL.
+  // :TODO: Remove once pixel endpoint is handling larger events again.
```
[the remark this answers is not in the document]
Comment (jmtaber129) [reply e2e/heapIgnore.spec.js]: I thought so, but the sentinel poll times out without this.

File: e2e/heapIgnore.spec.js (hunk 1 above)
Comment (jmtaber129) [reply e2e/heapIgnore.spec.js]: done

File: e2e/heapIgnore.spec.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (GrandPoohBear) [reply e2e/heapIgnore.spec.js]: Huh, weird.... is it possible that the change made it into the prod environment but we just didn't configure the local one that way?

**Pull request merged**